_posts/en/newsletters/2019-01-29-newsletter.md

```diff
@@ -0,0 +1,219 @@
+  wallet?]({{bse}}83070) It's easy to setup two copies of Bitcoin Core,
+  one on an offline computer as a cold wallet for storing private keys
+  and one on a networked computer for monitoring the wallet balance and
+  broadcasting transactions.  But how would you actually use [BIP174][]
+  Partially-Signed Bitcoin Transactions (PSBTs) to spend money using
+  these two wallets?  BIP174 author Andrew Chow explains.
+
+- [Why relay transactions from node to node---why not send them to
+  miners directly?]({{bse}}83054) It seems like the Bitcoin network could
+  use a lot less bandwidth if everyone just sent their transactions to
+  miners directly and then nodes only distributed blocks.  Pieter Wuille
+  explains why that would be bad for privacy and the health of the
+  network, plus why it wouldn't even save that much bandwidth.
+
+- [Why should miners hashing arbitrary nonces inspire trust in
+  transaction security?]({{bse}}83951)  When described as a simple
+  guessing game, Bitcoin's proof of work doesn't sound compelling, but
+  this answer from one of Bitcoin StackExchange's top 30 experts,
+  Chytrik, provides a simple analogy that captures the essence of proof
+  of work and how it helps keep Bitcoin transactions secure.
+
+Optech also congratulates and thanks Pieter Wuille, who this month
+became the [all-time top-voted contributor][top bse] to Bitcoin
+StackExchange.
+
+## Notable code changes
+
+*Notable code changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+and [libsecp256k1][libsecp256k1 repo].*
+
+- [Bitcoin Core #14955][] switches the Random Number Generator (RNG)
+  used from OpenSSL to Bitcoin Core's own implementation, although RNG
+  output gathered by Bitcoin Core is fed out to OpenSSL and then read
+  back in when the program needs strong randomness.  This moves Bitcoin
+  Core a little closer to no longer needing to depend on OpenSSL, as
+  that dependency has caused security issues in the past.  The PR
+  description and the code changes are very well documented for anyone
+  concerned about the safety of this change.
+
+- [Bitcoin Core #14353][] adds a new REST call
+  `/rest/blockhashbyheight/` for fetching the block in the current best
+  block chain based on its height (how many blocks after the Genesis
+  Block it is).
+
+- [Bitcoin Core #15193][] sets the `whitelistforcerelay` configuration
+  option to off by default.  When enabled, this option cause a node to
+  relay transactions from its manually whitelisted peers and clients
+  even if those transactions violate node policy or consensus rules.
+  This could cause the relaying node, rather than the origin node or
+  client, to be banned by its peers, so it's better to default to
+  turning this option off.  Developers are also asking anyone using this
+  feature to [contact][contact core] them so that they know it's not an
+  unused option that should be deprecated in the future.
```

Comment:
Should this be called out as an Action Item in the newsletter?

Reply:
My gut feeling says "no", but that could just be because I'm too lazy to change it.  Not only do I think this is a minor feature that is probably unused, but the request to contact a dev if you use the feature has also been added to the draft Bitcoin Core release notes, which is the file all sysadmins should be reading before upgrading production systems, so I think just putting a mention in the main newsletter body text is enough for us.

However, if anyone disagrees, I'm happy to put it in the action items.

Reply:
Agree with Dave. I don't think this is required as an action item.